### DISCUSSION TITLE
I am having an issue with Gatsby. I am new. I am using Windows.

### DISCUSSION THREAD

**Discussion by BillPinder** [gatsbyjs/gatsby#35013] [Help]
I am taking a course and I am instructed to make a new gatsby project like this:

gatsby new 'project-name' 'github URL'

I get the error 'git' is not recognized as an internal or external command,
ERROR
Command failed with exit code 1: git clone 'github URL above + .git' 'project name' --recursive --depth=1

There is more that I can include if you need it to help me. It's probably something super easy that I am missing.
Thanks.

**Comment by LekoArts** [comment]
Hi!

You'll want to install [Git](https://www.atlassian.com/git/tutorials/install-git#windows) on your computer, it's explained here: https://www.gatsbyjs.com/docs/tutorial/part-0/#installation-guide
